src/Stratis.Bitcoin.Features.Wallet.Tests/WalletManagerTest.cs

```diff
@@ -33,6 +33,92 @@ public WalletManagerTest(WalletFixture walletFixture)
             this.walletFixture = walletFixture;
         }
 
+        /// <summary>
+        /// Creates a wallet with valid trx and verifies that the ShellHelper is called for the found UTXO's.
+        /// </summary>
+		[Fact]
+		public void VerifyExecutionOfWalletNotify()
```

Comment:
@sondreb it looks like all the code changes were submitted with extra indentation?